Google-Summer-of-Code-2023.md

```diff
@@ -697,6 +699,49 @@ Additionally, finish adding [computer-aided translation (CAT)](https://docs.goog
 **Useful resources:**
 
 
+### 2.3. Improvements for translation reviewer experience
+
+**Project Description:**
+
+Translation reviewers have given feedback such that the current contributor dashboard reviewer workflow doesn’t match with most of their needs. This project aims to improve the translation reviewer experience in the contributor dashboard.
+
+**Size of this project:** large (~350 hours)
+
+**Potential Mentors:** TBA
+
+**Knowledge/Skills Recommended:**
+
+* Knowledge and understanding of Python
+* Knowledge and understanding of TypeScript, Angular, basic HTML, and CSS
+
+**Suggested Milestones:**
+
+* **Milestone 1:**
+  - Show translation cards according to the order of the content flow in the lesson.
+  - Link to the exploration editor so that the original lesson can be seen ([Mock](https://docs.google.com/document/d/1lIXkcQwPcPeS715vhX6mY-lowzltLxE8v2QQEvFadks/edit#heading=h.v0ksth5jphpg))
+  - Show the size of the translation ([Mocks in progress](https://github.com/oppia/design-team/issues/53))
+  - Allow translators to pin/unpin ([Mocks in progress](https://github.com/oppia/design-team/issues/53))
```

Comment:
Please see the previous comment I left, it hasn't been responded to or addressed:

> (one question -- if this is for translation reviewers then why is the fourth bullet point about translators ("pin/unpin specific lessons to translate")? May want to be clear which of these are for reviewers, translators, or both, and edit the project description accordingly. /cc @sagangwee @dchen97 )

Reply:
Yup, this should be translation reviewers. Thanks.

Reply:
changed translators -> translation reviewers